src/components/molecules/LiveSchedule/LiveSchedule.tsx

```diff
@@ -0,0 +1,86 @@
+import { useSelector } from "hooks/useSelector";
+import { useUser } from "hooks/useUser";
+import React, { FC } from "react";
+import { VenueEvent } from "types/VenueEvent";
+import { isEventLive } from "utils/event";
+import { currentTimeInUnixEpoch, formatHourAndMinute } from "utils/time";
+import { isExternalUrl } from "utils/url";
+import { enterRoom } from "utils/useLocationUpdateEffect";
+import "./LiveSchedule.scss";
+
+const LiveSchedule: FC = () => {
+  const { user, profile } = useUser();
+  const { venueEvents, venue } = useSelector((state) => ({
+    venueEvents: state.firestore.ordered.venueEvents,
+    venue: state.firestore.ordered.currentVenue[0],
+  }));
```

Comment:
As above RE: extracting selectors, particularly about avoiding this 'compound selector' style

Reply:
this is still an issue

Ideal fix:

Create the selectors outside of the component (eg. top of file, `src/utils/selectors.ts`, etc):

```typescript
const venueSelector = (state: RootState) => state.firestore.ordered.currentVenue[0]
const venueEventsSelector = (state: RootState) => state.firestore.ordered.venueEvents
```

Then you can do:
```suggestion
  const venue = useSelector(venueSelector)
  const venueEvents = useSelector(venueEventsSelector)
```

---

I believe this is one of the things I have refactored in https://github.com/sparkletown/sparkle/pull/721 / https://github.com/sparkletown/sparkle/pull/726 
Though don't let that block this PR. Fix as above, and I can worry about any fixes my branch will need later.